gui/src/main/index.ts

```diff
@@ -88,6 +88,8 @@ const GUI_VERSION = app.getVersion().replace('.0', '');
 /// Mirrors the beta check regex in the daemon. Matches only well formed beta versions
 const IS_BETA = /^(\d{4})\.(\d+)-beta(\d+)$/;
 
+const VERSION_CHECK_DISABLED = process.env.DISABLE_VERSION_CHECK === '1';
```

Comment:
In addition to the suggestion from faern, I would like to change the name a bit since it doesn't actually disable version checks. It just disables version checks initiated by the Electron app and ignores the result. How about something like `MULLVAD_DISABLE_UPDATE_NOTIFICATION`?